Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -17,14 +17,21 @@
 #include "Core/ConfigManager.h"
 #include "Core/Core.h"
 
+// OpenGL is not available on Windows-on-ARM64
+#if !defined(_WIN32) || !defined(_M_ARM64)
```

Comment:
this doesn't seem like a good way/place to check this (e.g. for some non-windows platform that wants to disable ogl). but i guess not a big deal.

Reply:
Agreed, we probably should move it to cmake (and make all backends selectable at compile time). But IMO that's a separate change.